README.md

```diff
@@ -15,12 +12,12 @@ and is inspired by [PineTime Hermes](https://github.com/Dejvino/pinetime-hermes-
 ## Features
 
 - [x] 100 % Free Software
-- [x] Low power consumption: 1.38 mA in idle mode (1.28 mA with the heart rate sensor disconnected), which gives a battery life of about one week.
```

Comment:
Could be nice to get the exact measurement sometimes on the master, to see in which direction it goes. 
Maybe in the wiki ?

Reply:
Yes, I think the wiki is a good place for that. I no longer have a way to measure the consumption of my device. I could fix that later on though.